dlc-test/src/test/scala/org/bitcoins/dlc/DLCClientTest.scala

```diff
@@ -468,18 +527,22 @@ class DLCClientTest extends BitcoinSAsyncTest {
 
       cetFailures = outcomes.map { outcome =>
         val oracleSig =
-          oraclePrivKey.schnorrSignWithNonce(outcome.bytes, preCommittedK)
+          oraclePrivKey.schnorrSignWithNonce(
+            CryptoUtil.sha256(outcome.asInstanceOf[EnumOutcome].outcome).bytes,
```

Comment:
ping

Reply:
I think I'm actually okay with this one in its test context